src/components/doctorForm/DoctorForm.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useRef } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { createDoctor } from '../../redux/doctor/doctorSlice';
+
+import './doctorform.css';
+
+const DoctorForm = () => {
+  const formRef = useRef();
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const formData = new FormData(formRef.current);
+    const name = formData.get('name');
+    const specialization = formData.get('specialization');
+    const bio = formData.get('bio');
+    const imageUrl = formData.get('image');
+
+    const data = {
+      doctor: {
+        name,
+        specialization,
+        bio,
+        image_url: imageUrl,
+      },
+    };
+    dispatch(createDoctor(data));
+    navigate('/doctors');
+  };
+
+  return (
+    <div className="container">
+      <h1 className="title">You can now create a doctor</h1>
+      <form className="form" ref={formRef} onSubmit={handleSubmit}>
+        <div className="form-input">
+          <label htmlFor="name">Your Full Name</label>
+          <input
```

Comment:
Great work create doctor functionality is working perfectly according to the requirements. Only a minor change I would suggest; As if the user is creating a doctor the text in line 35 should be "Create a Doctor" instead, also in line 38 instead of "Your full name" it should be something like "Doctor's Full Name".